modules/userManager.xqm

```diff
@@ -167,7 +167,7 @@ declare function usermanager:update-user($user-name as xs:string, $user-json as
                 else(),
 
                 (: if a password is provided, we always change the password, as we dont know what the original password was :)
-                for $group in secman:get-user-groups($user) return secman:remove-group-member($group, $user),
+                for $group in secman:get-user-groups($user)[. != sm:get-user-primary-group($user)] return secman:remove-group-member($group, $user),
```

Comment:
It might be best to stick with the unusual `secman` prefix defined in the prolog, or otherwise switch the whole module to the standard `sm`.

Reply:
yeah, it does get on my nerves and, as demonstrated, is easy to make a silly mistake